Assets/Scripts/Inventory/Chest.cs

```diff
@@ -0,0 +1,170 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Chest : PlacedObject
+{
+    public List<Item> contents = new List<Item>();
+
+    public GlobalStorage globalStorage;
+
+    public Vector3 coordinate;
```

Comment:
This location variable should eventually become obsolete as the parent class "PlacedObject" should contain coordinates. However, atm PlacedObject still needs some catching up so this is completely fine for now.